src/styles/block/pruduct-card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
```

Comment:
According to the code style rules, the padding should be written in the shorthand form when all four paddings are present. In this case, you're only specifying horizontal padding, so consider using `padding-left` and `padding-right` instead.